src/library/Uno.Material/Controls/ModalStandardBottomSheet.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Uno.Disposables;
+using Uno.Material.Entities;
+using Windows.UI.Xaml;
+
+namespace Uno.Material.Controls
+{
+	[TemplateVisualState(GroupName = __sheetStateGroup, Name = __closedState)]
+	[TemplateVisualState(GroupName = __sheetStateGroup, Name = __openedState)]
+	public partial class ModalStandardBottomSheet : StandardBottomSheet
```

Comment:
> ModalStandardBottomSheet: When swiped down, the gray overlay is still visible.

@NicoChampagne Lets continue the swipe down dismiss issue in this thread.
The issue can be reproduced on uwp, android at least. Instead of *fling it*, just drag and release at 10~20% from the edge with 0 momentum. The sheet will continue to slide offscreen, but leaving the overlay:
![20200909 ModalBotSheet swipe dismiss](https://user-images.githubusercontent.com/2359550/92642691-98cca080-f2ae-11ea-935a-368af1f29b5d.gif)

Reply:
@Xiaoy312  I see! As the sheet doesn't snap into position the current snap point is not updated, thanks for the details repro!

Reply:
fixed